library/src/app/components/bank-account-management/bank-account-connect/bank-account-connect.component.html

```diff
@@ -0,0 +1,92 @@
+<ng-container *ngIf="configService.getConfig$() | async as config">
+  <mat-card-title>
+    <app-cybrid-logo
+      [ngClass]="
+        config.theme === 'LIGHT' ? 'cybrid-logo-light' : 'cybrid-logo-dark'
+      "
+    ></app-cybrid-logo>
+  </mat-card-title>
+  <ng-container *ngIf="(mobile$ | async) !== true; else mobile">
+    <mat-card-content>
+      <!-- Error -->
+      <ng-container *ngIf="error$ | async; else content">
+        <div class="connect-content">
+          <div>
+            <mat-icon color="warn">cancel</mat-icon>
+            <strong>{{ 'unexpectedError' | translate }}</strong>
+          </div>
+        </div>
+      </ng-container>
+      <ng-template #content>
+        <mat-stepper #stepper linear="true" animationDuration="0">
+          <!-- Connect Step -->
```

Comment:
We should use partials for these steps

Reply:
This is a bit trickier that the identity-verification component. I've tried to pull the logic out but it doesn't result in it being easier to read, or really making it much shorter. I may make a general step partial that can function for any component going forward.